tests/conftest.py

```diff
@@ -1656,9 +1656,13 @@ def finalizer():
                     # We are allowing 20 re-tries for health check, to avoid teardown failures for cases like:
                     # "flip-flopping ceph health OK and warn because of:
                     # HEALTH_WARN Reduced data availability: 2 pgs peering
-                    ceph_health_check(
-                        namespace=ocsci_config.ENV_DATA["cluster_namespace"]
-                    )
+                    cluster_type = constants.CLUSTER_TYPE_INTERNAL
+                    if ocsci_config.DEPLOYMENT.get("external_mode"):
+                        cluster_type = constants.CLUSTER_TYPE_EXTERNAL
```

Comment:
Classic external mode deployment will run the same ceph health as of now, we should not do any changes here, just in case of multi cluster we should make sure all ceph health in this fixture cover both clusters health.

```suggestion
```

Reply:
There is missing also the ceph_health_check call on the line 1681 and 1691 to be updated to pass the cluster_type = constants.CLUSTER_TYPE_MULTI_SC

So I would move the cluster type definition on top level of fixture, and then you can just propagate it to all places.